tests/integration/testproofs.nim

```diff
@@ -131,55 +134,58 @@ marketplacesuite "Simulate invalid proofs":
+        # .withLogTopics("marketplace", "sales", "reservations", "node", "clock"),
 
     providers:
       CodexConfig()
         .nodes(1)
-        .simulateProofFailuresFor(providerIdx=0, failEveryNProofs=3)
+        .simulateProofFailuresFor(providerIdx=0, failEveryNProofs=1),
         # .debug() # uncomment to enable console log output
-        .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
-        .withLogTopics("marketplace", "sales", "reservations", "node"),
+        # .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
+        # .withLogTopics("marketplace", "sales", "reservations", "node"),
 
     validators:
       CodexConfig()
         .nodes(1)
         # .debug()
-        .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
-        .withLogTopics("validator", "onchain", "ethers")
+        # .withLogFile() # uncomment to output log file to tests/integration/logs/<start_datetime> <suite_name>/<test_name>/<node_role>_<node_idx>.log
+        # .withLogTopics("validator", "onchain", "ethers", "clock")
   ):
     let client0 = clients()[0].client
-    let totalPeriods = 25
+    let expiry = 5.periods
+    # In 2 periods you cannot have enough invalid proofs submitted:
+    let duration = expiry + 2.periods
 
-    let datasetSizeInBlocks = 2
-    let data = await RandomChunker.example(blocks=datasetSizeInBlocks)
-    createAvailabilities(data.len, totalPeriods.periods)
+    let data = await RandomChunker.example(blocks=8)
+    createAvailabilities(data.len * 2, duration) # TODO: better value for data.len
 
     let cid = client0.upload(data).get
 
     let purchaseId = await client0.requestStorage(
       cid,
-      expiry=10.periods,
-      duration=totalPeriods.periods,
-      origDatasetSizeInBlocks=datasetSizeInBlocks)
+      expiry=expiry,
+      duration=duration,
+      nodes=3,
+      tolerance=1,
+      proofProbability=1
+    )
     let requestId = client0.requestId(purchaseId).get
 
-    check eventually client0.purchaseStateIs(purchaseId, "started")
+    check eventually(client0.purchaseStateIs(purchaseId, "started"), timeout = expiry.int * 1000)
 
     var slotWasFreed = false
     proc onSlotFreed(event: SlotFreed) =
-      if event.requestId == requestId and
-          event.slotIndex == 0.u256:
+      if event.requestId == requestId:
         slotWasFreed = true
 
     let subscription = await marketplace.subscribe(SlotFreed, onSlotFreed)
 
     # check not freed
-    let currentPeriod = await getCurrentPeriod()
-    check not eventuallyP(slotWasFreed, currentPeriod + totalPeriods.u256 + 1)
+    await sleepAsync((duration - expiry).int.seconds)
+    check not slotWasFreed
 
     await subscription.unsubscribe()
 
```

Comment:
Whats missing here to be able to run the remaining tests?

Reply:
This was already commented out and I left it untouched, because it wasn't part of the prover integration. I want to clean up the integration tests with Eric and Adam after this is all merged, and this test will then either get fixed, or moved to a unit test.